src/devices_models/device_constructors/hydrogeneration_constructor.jl

```diff
@@ -41,7 +41,6 @@ function construct_device!(psi_container::PSIContainer, sys::PSY.System,
     reactivepower_variables!(psi_container, devices);
 
     #Constraints
-    activepower_constraints!(psi_container, devices, model, S, model.feed_forward)
     reactivepower_constraints!(psi_container, devices, model, S, model.feed_forward)
```

Comment:
Do we need to get rid of reactive constraints too?

Reply:
NO! reactive power is unrelated to active power and energy.